CSS/style.css

```diff
@@ -556,7 +552,7 @@ form {
 /* ==========================================contact-form========================================================== */
 .contact-section {
   width: 375px;
-  height: 775px;
+  height: 100vh;
```

Comment:
- [ ] Kindly, make sure your error message is visible for all screen size. You can check in the image below (and try yourself on different device mobile or desktop) and notice that the message is not visible in most of device (or screen size). You can fixe this by removing this line.

**You project:** error message is not visible and we can't scroll down:
![image](https://user-images.githubusercontent.com/74411135/233454697-7daea2f4-3824-4f86-b63c-17c41ab415cd.png)

This will make your project way more prettier which will improve the user experience 📱  😄 👍